diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index e0f6ac0..ef167fb 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -27,7 +27,7 @@ message(STATUS "GWOLLUM libraries: ${GWOLLUM_LIBRARY_DIRS}")
 message(STATUS "HDF5 libraries: ${HDF5_INCLUDE_DIRS}")
 
 # add link paths
-link_directories(${FRAMEL_LIBRARY_DIRS} ${GWOLLUM_LIBRARY_DIRS})
+link_directories(${FFTW_LIBRARY_DIRS} ${FRAMEL_LIBRARY_DIRS} ${GWOLLUM_LIBRARY_DIRS})
 
 # versioning
 configure_file(Oconfig.h.in ${CMAKE_CURRENT_BINARY_DIR}/Oconfig.h @ONLY)
@@ -90,6 +90,7 @@ target_link_libraries(
   ${ROOT_Hist_LIBRARY}
   ${ROOT_MathCore_LIBRARY}
   ${ROOT_RIO_LIBRARY}
+  ${FFTW_LIBRARIES}
   CUtils
   Inject
   RootUtils
